internal/pkg/addon/package.go

```diff
@@ -273,8 +281,129 @@ type s3Object struct {
 }
 
 func (a *Addons) uploadAddonAsset(assetPath string, forceZip bool) (s3Object, error) {
+	// make path absolute from wsPath
+	if !path.IsAbs(assetPath) {
+		assetPath = path.Join(a.wsPath, assetPath)
```

Comment:
For our windows users <img src="https://pbs.twimg.com/media/EF-36wfWwAEtTNY.jpg" data-canonical-src="https://pbs.twimg.com/media/EF-36wfWwAEtTNY.jpg" width="50" height="25" /> 
```suggestion
		assetPath = filepath.Join(a.wsPath, assetPath)
```